python/google/protobuf/internal/json_format_test.py

```diff
@@ -404,16 +596,13 @@ def testParseBadEnumValue(self):
 
   def testParseBadIdentifer(self):
     self.CheckError('{int32Value: 1}',
-                    (r'Failed to load JSON: Expecting property name'
-                     r'( enclosed in double quotes)?: line 1'))
```

Comment:
Er... we may not want to do up-integrate right now which will cause a lot of work.
Can you manually revert this kind of changes? There are 3 small changes you may want to take case of for this file:
https://github.com/google/protobuf/commits/master/python/google/protobuf/internal/json_format_test.py